shell/common/switches.cc

```diff
@@ -78,13 +80,13 @@ void PrintUsage(const std::string& executable_name) {
   std::cerr << std::string(column_width, '-') << std::endl;
 }
 
-const char* FlagForSwitch(Switch swtch) {
+const ftl::StringView FlagForSwitch(Switch swtch) {
```

Comment:
Actually the problem is that the result of `FlagForSwitch` is used as an input to several FTL APIs (e.g. `command_line.GetOptionValue`), and these APIs require a `StringView`. If you want to keep `char*`, I can create `StringView` objects only for the purpose of calling the FTL APIs.